toolkit/dump_tasks.php

```diff
@@ -44,16 +44,17 @@
 		exit - 1;
 	}
 
+	echo "iTop $sITopUrl, ";
 	if (empty($aResult['objects'])) {
-		echo "There is no SynchroDataSource defined on the iTop server ($sITopUrl).\n";
+		echo "no SynchroDataSource defined.\n";
 	} else {
-		switch (count($aResult['objects'])){
+		switch (count($aResult['objects'])) {
 			case 1:
-				echo "There is 1 SynchroDataSource defined on the iTop server ($sITopUrl):\n";
+				echo "1 SynchroDataSource defined:\n";
 				break;
 
 			default:
-				echo "There are ".count($aResult['objects'])." SynchroDataSource defined on the iTop server ($sITopUrl):\n";
+				echo count($aResult['objects'])." SynchroDataSource defined:\n";
```

Comment:
```suggestion
				echo count($aResult['objects'])." SynchroDataSource's defined:\n";
```

Otherwise this switch and case statement is not needed anymore..

Reply:
Oh you're right ! 
But why the apos ?
As it is unusual to make a plural out of an iTop object class name, I would suggest "... SynchroDataSource objects defined". What do you think ?

Reply:
That’s fine as well…

Reply:
Done in 1fed14f1